go/kubebuilder/memcached-operator/README.md

```diff
@@ -69,6 +81,17 @@ NAME                                                               DESIRED   CUR
 replicaset.apps/memcached-operator-controller-manager-864f7c75d4   1         1         1       118s
 ```
 
+### Verify Webhooks
+```
+$ kubectl patch memcached memcached-sample -p '{"spec":{"size": 4}}' --type=merge -n memcached-operator-system
+```
+Above command to increase the pod size to even number should throw error as shown below, as the validation webhook does not allow even number pods.
+
+```
+Error from server (Cluster size must be an odd number): admission webhook "vmemcached.kb.io" denied the request: Cluster size must be an odd number
+```
+
```

Comment:
````suggestion
### Verifying the validating webhook

The following command attempts to increase the CR's `spec.size` to an even number. It should throw an error, like that shown below, as the validating webhook does not allow an even `spec.size`.

```console
$ kubectl patch memcached memcached-sample -p '{"spec":{"size": 4}}' --type=merge -n memcached-operator-system
Error from server (Cluster size must be an odd number): admission webhook "vmemcached.kb.io" denied the request: Cluster size must be an odd number
```
````